change/@microsoft-teams-js-504be7c5-dd1e-4992-9779-fe2836fa75c1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "create raiseHandStateChangedHandler",
```

Comment:
Please be more descriptive since this will get turned into CHANGELOG entries used by developers upgrading their version of the sdk. Something like "Added meeting.registerRaiseHandStateChangedHandler"